tests/js/web/NavMainComponentTest.js

```diff
@@ -494,4 +500,21 @@ describe('NavMainComponent', function () {
             expect(this.$navQuaternary.hasClass('is-open')).to.be.false;
         });
     });
+
+    describe('opening the secondary nav when a popover is open', function () {
+        beforeEach(function () {
+            this.navMainComponent.init();
+            this.clickEvent = $.Event('click');
+            this.$linkOne.trigger(this.clickEvent);
+            this.$popoverLink.trigger(this.clickEvent);
```

Comment:
Are we sure this bit works? I've had issues in the past with triggering the same click event on multiple elements in the same beforeEach.

Reply:
Worked for me, and fails when I comment out the related code. So.... yeah?